app/metrics.yaml

```diff
@@ -3131,3 +3131,72 @@ perf.awesomebar:
       - fenix-core@mozilla.com
       - gkruglov@mozilla.com
     expires: "2020-10-01"
+
+progressive_web_app:
+  homescreen_tap:
+    type: event
+    description: |
+      A user taps on PWA homescreen icon
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/10261
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/issues/10261
+    notification_emails:
+      - fenix-core@mozilla.com
+      - erichards@mozilla.com
+    expires: "2020-09-01"
+  install_menu_tap:
+    type: event
+    description: |
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/10261
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/issues/10261
+    notification_emails:
+      - fenix-core@mozilla.com
+      - erichards@mozilla.com
+    expires: "2020-09-01"
+  add_to_homescreen_tap:
+    type: event
+    description: |
+      A user taps the add to homescreen icon for a PWA
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/10261
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/issues/10261
+    notification_emails:
+      - fenix-core@mozilla.com
+      - erichards@mozilla.com
+    expires: "2020-09-01"
+  foreground:
+    type: event
+    description: |
+      A user brings the PWA into the foreground.
+    extra_keys:
+      duration:
+        description: |
+          The current time in ms when the PWA was brought to the foreground.
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/10261
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/11859
+    notification_emails:
+      - fenix-core@mozilla.com
+      - erichards@mozilla.com
+    expires: "2020-10-01"
+  background:
+    type: event
+    description: |
+      A user puts the PWA into the background.
```

Comment:
This should be adjusted with `foreground` to indicate that it doesn't refer to the PWA, just the intent receiver activity.